edx_proctoring/backends/tests/test_rest.py

```diff
@@ -229,11 +230,23 @@ def test_get_javascript_bundle(self, get_files_mock):
         self.assertEqual(javascript_url, '/there/it/is')
 
     @patch('edx_proctoring.backends.rest.get_files')
-    def test_get_javascript_empty_bundle(self, get_files_mock):
+    def test_get_javascript_empty_return(self, get_files_mock):
         get_files_mock.return_value = []
         javascript_url = self.provider.get_javascript()
         self.assertEqual(javascript_url, '')
 
+    @patch.object(webpack_loader.loader.WebpackLoader, 'get_assets')
```

Comment:
You don't have to import the object in order to patch it. `@patch('webpack_loader.loader.WebpackLoader.get_assets')` should work.